hw/dv/tools/vcs/vcs.tcl

```diff
@@ -0,0 +1,26 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# TCL file invoked from VCS's simv at run-time using this: -ucli -do <this file>
```

Comment:
This is completely identical to `xcelium.tcl`.  wonder if we can consolidate further.

Reply:
Yeah, I couldn't decide, but thought that we were likely to want to add something tool-specific at some point and this gives a handy place to put it. Would you prefer them to be merged into a script in `dv/tools`?

Reply:
This is fine for now. We will worry about it later.